python3/plugins/extauth-hook-AD.py

```diff
@@ -265,16 +239,15 @@ def _add_upn(self, subject_rec):
 
     def _add_subject(self, subject_rec):
         try:
+            # ._format_item(name) would have formatted the name to be in the format
```

Comment:
.format_item is removed, so no need to comment it.
BTW, BPIS-->PBIS

Reply:
Thanks, removed.